Automattic/vip-go-mu-plugins
# Sanitize query before logging  'es_warning' log entries

## Description

We should always sanitize the query object when logging as it contains sensitive information. 

This PR fixes a particular occurrence when a response is a Warning from ES Server

## Changelog Description

### Plugin Updated: Enterprise Search

We've addressed an issue where potentially sensitive information may have been logged when the Elasticsearch server returns a `Warning` response. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

The issue happens particularly around the deprecation warning during `health validate-contents` command.

1. Apply the PR. 
2. Kick off `wp vip-search health validate-contents`.
3. Watch the logs and verify the sensitive part is removed.
